app/routines/update_user_salesforce_info.rb

```diff
@@ -143,7 +198,7 @@ def notify_errors
     return if @errors.empty?
     Rails.logger.warn("UpdateUserSalesforceInfo errors: " + @errors.inspect)
 
-    if @allow_error_email && is_real_production?
+    if @allow_error_email && Rails.application.is_real_production?
```

Comment:
Maybe it would be a good idea to change this to something specific in the secrets file, like:

```
salesforce:
  enable_exception emails: true
```

or something